ldap/docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+services:
+    ldap:
+        image: osixia/openldap:1.2.3
+        hostname: ldap
+        container_name: ldap
+        environment:
+            LDAP_ORGANISATION: "Confluent"
+            LDAP_DOMAIN: "confluent.io"
+        ports:
+            - "389:389"
+            - "636:636"
+        volumes:
+            - "$PWD/custom:/container/service/slapd/assets/config/bootstrap/ldif/custom"
+        command: "--copy-service"
+
+    zookeeper:
+        build: zookeeper/
+        hostname: zookeeper
+        container_name: zookeeper
+        environment:
+          - KAFKA_OPTS=-Djava.security.auth.login.config=/etc/kafka/zookeeper_server_jaas.conf
+        ports:
+            - "2181:2181"
```

Comment:
This port mapping is not strictly required and could create conflict if there is already a running ZK on the host. I would propose to remove it. (that's of course debatable, but as I have a ZK on my host, I hit this issue)